python/py_object.cpp

```diff
@@ -36,4 +36,6 @@ bool PyObject::operator==(const PyObject &other) const {
   return to_pybind().equal(other.to_pybind());
 }
 
+std::string to_string(const PyObject &obj) { return py::str(obj.to_pybind()); }
```

Comment:
Should this acquire the GIL first? `py::str` might segfault otherwise?

Reply:
Good point